draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1166,10 +1166,9 @@ In this example the bias is negligible, but in general it can be significant.
 To control bias, the input msg should be hashed to an integer comprising at
 least ceil(log2(p)) + k bits; reducing this integer modulo p gives bias at
 most 2^-k, which is a safe choice for a cryptosystem with k-bit security.
-To obtain such an integer, hash H with b-bit output should be evaluated W =
-ceil((ceil(log2(p)) + k) / b) times and the results concatenated to produce a
-(W * b)-bit integer. For example, for H = SHA256, k = 128-bit security, and p
-a 256-bit prime, W = ceil((256 + 128) / 256) = 2.
+To obtain such an integer, we use HKDF {{!RFC5869}} to expand the input msg
+with additional information (a counter) to a (ceil(log2(p)) + k)-bit string,
+which is then converted to an integer via OS2IP.
```

Comment:
I figured keeping "counter" (rather than a reference to `car`) as a parenthetical was enough for the curious, though you're right in that it's unnecessary information at this point. I'm fine with the example. I just took it out since it's a pretty trivial computation.